packages/components-js/tests/e2e/puppeteer/specs/carousel.e2e.ts

```diff
@@ -841,19 +862,19 @@ describe('activeSlideIndex', () => {
     expect(await isElementCompletelyInViewport(slide3)).toBe(false);
 
     await page.keyboard.press('Tab');
-    await waitForSlideToBeActive(slide2);
+    await waitForSlideToBeActive(slide1);
 
-    expect(await isElementCompletelyInViewport(slide1)).toBe(false);
+    expect(await isElementCompletelyInViewport(slide1)).toBe(true);
```

Comment:
Is this desired, since it behaves now differently than before?

Reply:
Yes, because keyboard navigation functionality was also optimized which affected slide movement.